CHANGES.rst

```diff
@@ -52,7 +52,8 @@ Bugfixes
 Internal Changes
 ^^^^^^^^^^^^^^^^
 
-- Nothing so far
+- Send signal `before_notification_send` before custom email to registrants (:pr:`5161`, thanks
+   :user:`vtran`)
```

Comment:
```suggestion
- Send signal `before_notification_send` before custom email to registrants (:pr:`5161`,
  thanks :user:`vtran`)
```

Reply:
Actually since it's not a new signal but just sending an existing one in another place I'm not sure if we really need a changelog entry for it...

Reply:
Well, I was just following indication for PR upstream. Up to you to decide if you want to keep that mention or not.